cmd/otel-agent/command/command.go

```diff
@@ -0,0 +1,64 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package command implements the top-level `otel-agent` binary, including its subcommands.
+package command
+
+import (
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/DataDog/datadog-agent/cmd/otel-agent/subcommands"
+	"github.com/DataDog/datadog-agent/cmd/otel-agent/subcommands/run"
+	"github.com/DataDog/datadog-agent/pkg/cli/subcommands/version"
+	"github.com/DataDog/datadog-agent/pkg/config/setup"
+)
+
+const (
+	// loggerName is the application logger identifier
+	loggerName = "OTELCOL"
+)
+
+// defaultConfigPath specifies the default configuration file path for non-Windows systems.
+var defaultConfigPath = filepath.Join(setup.InstallPath, "etc/otel-config.yaml")
```

Comment:
Is this configuration specific of Datadog? Or it is the OTel configuration? Mostly asking to ensure we understand the implications. 

1. If it DD specific. Could we use `datadog.yaml`?
2. If it is OTel specific I believe there is not much control from our side 😄

Reply:
This is the OTel configuration. And as you suspect, we indeed have to support native OTel configurations so that our "OTel Agent" can serve as a drop-in replacement to a customer's OTel Collector.

Reply:
OTel Agent is going to read collector config. In the spirit of small PRs, this PR still builds `config.Component` from datadog.yaml. In next PRs, we build config.component from Collector config.

Reply:
> In next PRs, we build config.component from Collector config.

😨 Is that something could discuss. I'm not sure 100% we want to include that as part of the config component. 

Let me know you are open to have a chat about it 😄

Reply:
We will build config component within OTel agent from collector config. It wouldn't be part of Config component module.  I did something similar here https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31669/files#diff-2c8297daaa3d686b1c4aaea155becaa75d23c1a65ad2e7b7ef07787c8ac92b0dR35